# Using -std=c11 lets the build fail with very weird errors (with icc). Switching to
# gnu11 makes it pass.
# Ward Poelmans <wpoely86@gmail.com>
diff -ur cmake-3.6.1.orig/Modules/Compiler/Intel-C.cmake cmake-3.6.1/Modules/Compiler/Intel-C.cmake
--- cmake-3.6.1.orig/Modules/Compiler/Intel-C.cmake	2016-07-22 15:50:22.000000000 +0200
+++ cmake-3.6.1/Modules/Compiler/Intel-C.cmake	2016-08-31 17:32:38.594401777 +0200
@@ -15,8 +15,8 @@
 endif()

 if (NOT CMAKE_C_COMPILER_VERSION VERSION_LESS 15.0.0)
-  set(CMAKE_C11_STANDARD_COMPILE_OPTION "${_std}=c11")
-  set(CMAKE_C11_EXTENSION_COMPILE_OPTION "${_std}=c11")
+  set(CMAKE_C11_STANDARD_COMPILE_OPTION "${_std}=gnu11")
+  set(CMAKE_C11_EXTENSION_COMPILE_OPTION "${_std}=gnu11")
 endif()

 if (NOT CMAKE_C_COMPILER_VERSION VERSION_LESS 12.1)
